ParisBNF/abb/BNFabb30.xml

```diff
@@ -0,0 +1,720 @@
+                    </history>
+                    <additional>
+                       <adminInfo>
+                          <recordHist>
+                             <source>
+                                <listBibl type="catalogue">
+                                   <bibl>
+                                      <ptr target="bm:ContiRossini1914Notice"/>
+                                      <citedRange unit="page">36-37</citedRange>
+                                      <citedRange unit="number">11</citedRange>
+                                      </bibl>
+                                   <bibl>
+                                      <ptr target="bm:Chaine1912Catalogue"/>
+                                      <citedRange unit="page">19</citedRange>
+                                      <citedRange unit="number">30</citedRange>
+                                   </bibl>
+                                   <bibl>
+                                      <ptr target="bm:Abbadie1859Cat"/>
+                                      <citedRange unit="page">35-37</citedRange>
+                                      <citedRange unit="number">30</citedRange>
+                                   </bibl>
+                                     </listBibl>
+
+                             </source>
+                          </recordHist>
+                       </adminInfo>
+                    </additional>
+                </msDesc>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+            <xi:fallback>
+          </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <particDesc>
+               <listPerson>
+                  <person>
+                     <persName ref="PRS1071dAbbadi" role="bequeather"/>
+                  </person>
+                  <person>
+                     <persName ref="PRS1071dAbbadi" role="owner"/>
+                  </person>
+                  <person>
+                     <persName ref="PRS12914Ahwan" role="owner"/>
+                     </person>
+
+                <person>
+                   <persName ref="PRS11012Gad" role="scribe"/>
+                   </person>
+
+                   <person>
+                      <persName ref="PRS11012Gad" role="scribe"/>
+                      </person>
+
+
+                </listPerson>
+                </particDesc>
+            <textClass>
+                <keywords>
+                    <term key="Bible"/>
+                    <term key="ChristianLiterature"/>
+                    <term key="OldTestament"/>
+                    <term key="Religion"/>
+                    <term key="Gon"/>
```

Comment:
would it make sense to add BetaEsrael? It once belonged to a member of the B.E. community, but was not produced by them?

Reply:
It does make sense; the ms only passed through the hands of a BetaEsrael fellow (according to d'Abbadie),  but among the keywords i see only BetaEsraelLiterature, which is not what you mean, right?

Reply:
after discussion with @SophiaD-M , the use of the BetaEsraelLiterature would be definitely warranted here, since this manuscript seems to be a typical specimen of manuscripts, even if produced by Christian scribes, "made" Beta esrael through their use of the community.

Reply:
The problem is that there is no specific traces of the use of this ms by the Beta esrael community whatsoever (also the later historiographical addition is included probably by a Christian); the information on the owner is added in a secondary hand, possibly by d'Abbadie himself. Probable is only that the last owner - who sold it to d'Abbadie - was a Beta esrael person.

Reply:
From Sophias forthcoming article 

> Aešcoly had another understanding of such practices, and states that the Betä Ǝsraʾel are so looked after that some people were tempted to sell fake, he calls it ‘pseudo-falacha’, manuscripts as their own. As an example he gives Paris, BnF, d’Abbadie 30, which “portant en tête la formule eulogique falacha, et qui n’est pourtant qu’un ouvrage chretien.”  This is only half true as the contained texts are all Old Testament material, plus Henok, except for the short Christian prayers in the fly leaves and a chronological text on the last folia). Next to the Betä Ǝsraʾel doxology of which was added by a later hand, the same hand wrote “this book belongs to abba Aḥwän”. From d’Abbadie’s catalogue description we learn that he bought the manuscripts directly from the Betä Ǝsraʾel abba Aḥwän.  Contrary to Aešcoly’s opinion I would call this a perfectly fine Betä Ǝsraʾel manuscript. It is part of the group of very large de luxe codices (see below), prepared by Christian scribes (note the ornamented leather covers), but somehow repurposed to suit the Betä Ǝsraʾel. These manuscripts often also follow other typical “Christian” codicology rules such as, stichometry,  or textile threads as bookmarks.

If we understand BetaEsraelLiterature as meaning "Manuscripts related to b.e.", it would be appropriate. This might be usable approach, because there are in fact very few decidedly B.E. texts. What do you think?

Reply:
To call this Beta esrael manuscript would be for me an overinterpretation.  Old Testament material is Jewish/Christian, Henok is Jewish/Christian (well, we should not move the focus, because the entire Christianity stands on the Jewish foundations) and has a high authority in Christian Ethiopia; "the short Christian prayers" are: 1) a note on tributes, probably related to a church (difficult to read, but most probably from the Christian context); 2) the full Mälke' hymn to Four Anymals (not imaginable for Beta esrael context - they would took it out, it is on the flyleaves). The names of the commisioners are perfectly Christian. There is really only one indication to Beta esra'el: the crudely written ownership note above, f. 2r,  with the name Ahwan (never heard, one could try to trace its origin, a task for Sophia!), and the information by d'Abbadie that the fellow was a Falasha monk. This information only indicates that the manuscript was in his hands for a while, and in my opinion, nothing more. 
For me, it is not a deluxe manuscript, it is rather a middle standard of the Gondarine time. There is another feature that can be traced: a very short introductory formulas, indeed, without "ba-sema ab wa-wald wa-manfas qeddus", this is true, but one has to check if is the general feature of the OT mss.